qa-tests-backend/src/test/groovy/VulnMgmtSACTest.groovy

```diff
@@ -240,9 +240,9 @@ class VulnMgmtSACTest extends BaseSpecification {
     }
 
     // GraphQL does not provide ordering guarantees, so to compare the results
-    // of two GraphQL queries we extract just the CVE names and sort them.
-    def extractCVEsAndSort(queryCallResult) {
-        return queryCallResult.results*.cve.sort()
+    // of two GraphQL queries we extract just the CVE names as a Set.
+    Set<String> extractCVEs(queryCallResult) {
+        return (queryCallResult.results*.cve.toSet() + UUID.randomUUID().toString()) as Set<String>
```

Comment:
I suppose the extra UUID.randomUUID().toString() is there in order to have a failing test and validate the error reporting code.

Reply:
Correct, I'll revert it once I see it's working correctly